test/tests/problems/reference_residual_problem/tests

```diff
@@ -0,0 +1,26 @@
+[Tests]
```

Comment:
Your requirements still aren't very good at meeting IEEE recommendations. Try to remove the implementation detail when writing the requirement. Assume that the requirement came first so object names and those types of things aren't assumed. This makes it more difficult as your requirement becomes more wordy to try to setup up all of the criteria for making the test unique, but that's precisely the point.

Reply:
Ok, avoiding the implementation details makes sense to me 👍 . But I can't get over using "the system" when the number one requirement for a requirement is that it be unambiguous.

Reply:
Ok, just read your response [here](https://github.com/idaholab/moose/pull/14004#discussion_r321797306). Sounds good!